README.md

```diff
@@ -23,5 +24,6 @@ Currently this is a manual process.
 1. Execute `npm run build` to build the zip
 1. Make a GitHub Release with Tag of `v<RELEASE NUMBER>`, adding release notes
 1. Add the zip to the release
+1. Add the Categories PDF to the release
```

Comment:
Yes it makes sense it's a separate file in the GH release so it can be downled and viewed separately. However it is bigger than the ZIP! It would be useful to add it to the ZIP as well.

Reply:
> It would be useful to add it to the ZIP as well.

I've now updated the build order and `mkzip.js` to include the PDF.